CHANGELOG.md

```diff
@@ -2,10 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## 3.0.0 (under development)
+## 3.1.0 (under development)
+
+## 3.0.0 (2018-01-10)
 
 - Add support for auto-generated DSL and Job DSL extensions by executing the Job DSL scripts in a local Jenkins
   instance.
+- PipelineBuilder: Do not overwrite values with default configuration.
```

Comment:
How about referring to issue numbers (if any) here?

Reply:
Done.

Reply:
Uhm, and have links? 😉

Reply:
Aren't ticket numbers automatically converted to links?
https://help.github.com/articles/autolinked-references-and-urls/#issues-and-pull-requests

Reply:
> References to URLs, issues, pull requests, and commits are automatically shortened and converted into links.

So, not in `README.md`.

Reply:
You are right. I will improve this in another commit later and merge this now so that you can rebase your other commit.